README.md

```diff
@@ -1,16 +1,66 @@
-# errors Integration Pack
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-## Configuration
-TODO: Describe configuration
+# Errors Pack
 
+# Introduction
+This pack holds all of the contains all actions and workflows needed for finding and formating
```

Comment:
wording is a little messed up here. Lets give some information about how we get the error message and tree using the st2 api (python module)